### DISCUSSION TITLE
Regarding HW5 gradings

### DISCUSSION THREAD

**Discussion by dhruvpatel-9** [divyagiridhar/CSC510_Group25_HW02#35] [General]
The group numbers shown in the Discord channel, the groups which we have to grade, and the links provided of the groups in the google sheet are different, what should we do?

**Comment by divyagiridhar** [comment]
I searched for the group number on the discord channel and their github repo

**Comment by divyagiridhar** [comment]
Also, are we supposed to evaluate later tomorrow as the groups will make changes

**Comment by dhruvpatel-9** [comment]
Exactly, need to ask the TA, also, there is a google sheet which provides the links for the groups we need to grade, some of the links are different in accordance to the group numbers which we have been assigned
